src/session.ts

```diff
@@ -160,8 +160,9 @@ export class MediaSession {
         options: Options = {},
     ): MediaEvent {
         // Set event option based on options or current state
-        this.currentPlayheadPosition =
-            options?.currentPlayheadPosition || this.currentPlayheadPosition;
+        if (options?.currentPlayheadPosition !== undefined) {
+            this.currentPlayheadPosition = options?.currentPlayheadPosition;
+        }
```

Comment:
Actually, since this is TS, i think you can do ?? instead of ||.  
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or

Try that and see.  Yay ts!

Reply:
@rmi22186 - tested the suggested change and confirmed its working and pushed the new code, thanks!